tests/buzzn_test_case.py

```diff
@@ -13,12 +13,18 @@ class TestConfig():
     CLIENT_NAME = 'BuzznClient'
     SQLALCHEMY_TRACK_MODIFICATIONS = False
     SQLALCHEMY_DATABASE_URI = 'sqlite:///:memory:'
-    PASSWORD_SALT = bcrypt.gensalt()
-
+    PASSWORD_SALT = bcrypt.gensalt().hex()
+    BUZZN_SMTP_SERVER = 'BUZZN_SMTP_SERVER'
+    BUZZN_SMTP_SERVER_PORT = '456'
+    BUZZN_EMAIL = 'team@buzzn.net'
+    BUZZN_EMAIL_PASSWORD = 'BUZZN_EMAIL_PASSWORD'
+    BUZZN_BASE_URL = 'localhost:5000'
```

Comment:
What are the effects of this line?

Reply:
`Baseurl of the backend i.e. address. Everyhing which needs to be in front of the api calls. Example mybuzzn-backend.buzzn.net` I also added this to the readme :)